android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -83,57 +169,94 @@ public void maybeAttachAdView(final String placement, final String customData, f
             return;
         }
 
-        // Run in next run loop when view is laid out and `getWidth()` / `getHeight()` has appropriate values
-        post( new Runnable()
+        if ( TextUtils.isEmpty( adUnitId ) )
+        {
+            AppLovinMAXModule.e( "Attempting to attach MaxAdView without Ad Unit ID" );
+            return;
+        }
+
+        if ( adFormat == null )
+        {
+            AppLovinMAXModule.e( "Attempting to attach MaxAdView without ad format" );
```

Comment:
Actually - these should not be error logs since it's a legit use case when we attempt to mount without ad unit id or ad format provided _yet_.

Reply:
Right, moved inside the runnable on both android and ios.